### ISSUE TITLE
`pure_pulse` in `CyclicNoiseGen_v1` is not pure pulse.

### ISSUE THREAD

**Issue by skitaoka** [nii-yamagishilab/project-NN-Pytorch-scripts#36]
`pure_pulse` is not pure pulse in [this line](https://github.com/nii-yamagishilab/project-NN-Pytorch-scripts/blob/3ae138cdd1f9e77d74f492181c14ca954e69ff3f/project/01-nsf/cyc-noise-nsf-4/model.py#L703), because the `noise` is not simple additive noise ([the added noise is masked](https://github.com/nii-yamagishilab/project-NN-Pytorch-scripts/blob/3ae138cdd1f9e77d74f492181c14ca954e69ff3f/project/01-nsf/cyc-noise-nsf-4/model.py#L653)). 

Is this behavior intended?

**Comment by TonyWangX** [comment]
Hi, thanks for pointing it out : )
I will check it during the weekends.

**Comment by TonyWangX** [comment]
Hi @skitaoka 

Thank you for looking into the implementation. 

You are correct, the `pure_pulse` has small noises in voiced regions. 

I think a better implementation should be like this:
#38 

https://github.com/nii-yamagishilab/project-NN-Pytorch-scripts/pull/38/files

 
Please have a check : )

**Comment by TonyWangX** [comment]
Here are two samples, one before revising the code, the other after.

Both used the pre-trained model in 00_demo.sh.

[samples.zip](https://github.com/nii-yamagishilab/project-NN-Pytorch-scripts/files/9934703/samples.zip)

**Comment by skitaoka** [comment]
great! :+1:
